From 518135b0cd6878e91e5fcf489f3185740ec307c0 Mon Sep 17 00:00:00 2001
From: Carlos Rafael Giani <dv@pseudoterminal.org>
Date: Sun, 12 Oct 2014 19:31:47 +0200
Subject: [PATCH] Remove hard coded values for CC and CXX

In spite of what the comment says, the environment variables do *not*
override the variables; "which which arm-linux-gnueabihf-gcc" etc. are
always called. To undo this, remove the change.

Upstream-Status: Pending

The default values cannot be overriden without the patch; upstream
might be interested in this.

Signed-off-by: Carlos Rafael Giani <dv@pseudoterminal.org>
---
 build/common.gypi | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/build/common.gypi b/build/common.gypi
index 7ec5ae2..11e8d1d 100644
--- a/build/common.gypi
+++ b/build/common.gypi
@@ -6047,16 +6047,6 @@
         ['CXX.host', '<(host_cxx)'],
       ],
     }],
-    ['OS=="linux" and target_arch=="arm" and host_arch!="arm" and chromeos==0 and clang==0', {
-      # Set default ARM cross tools on linux.  These can be overridden
-      # using CC,CXX,CC.host and CXX.host environment variables.
-      'make_global_settings': [
-        ['CC', '<!(which arm-linux-gnueabihf-gcc)'],
-        ['CXX', '<!(which arm-linux-gnueabihf-g++)'],
-        ['CC.host', '<(host_cc)'],
-        ['CXX.host', '<(host_cxx)'],
-      ],
-    }],
     # TODO(yyanagisawa): supports GENERATOR==make
     #  make generator doesn't support CC_wrapper without CC
     #  in make_global_settings yet.
-- 
1.9.1

